recipes/pytfa/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "pytfa" %}
+{% set version = "0.9.3" %}
+{% set build_number = '0' %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/EPFL-LCSB/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  #url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b4d848bd04cf8384b05e82fecb25a7cc59a65031ac5dee2c732853a12b086014
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+
+  run:
+    - python >=3.8
+    - equilibrator-api
+    - equilibrator-cache
+    - sympy <=1.5
+    - networkx
+    - pandas
+    - cobra <=0.24.0
+    - bokeh >=0.12.1
+    - numpy
+    - optlang
+    - scipy
+    - tqdm
+    - python-libsbml
+
+test:
+  imports:
+    - pytfa
+
+# CURRENTLY DEPRECATED BECAUSE OF GIT ARCHIVE PROBLEM
+#  commands:
+#    - pytest
+#  # declare any test-only requirements here
+#  requires:
+#    - pytest
+#  # copy over any needed test files here
+#  source_files:
+#    - tests
+#    - data
+#    - models
```

```suggestion
test:
  imports:
    - pytfa
  commands:
    - pip check
  requires:
    - pip
```
Think of this test section as *packaging* tests. For noarch python packages, the correctness tests are the responsibility of upstream and since pure python packages shouldn't have linking errors other compiling related errors. `pip check` and import tests should be sufficient because were are just testing that all of the dependencies are correct and that the package is importable. (We would also test a CLI if the package has one, but this package does not.)

---

Ok I see! Thank you for this explanation this is very helpful :)